crypto/src/main/scala/io/iohk/ethereum/crypto/package.scala

```diff
@@ -41,6 +41,15 @@ package object crypto {
   def kec256(input: ByteString): ByteString =
     ByteString(kec256(input.toArray))
 
+  def kec256PoW(header: Array[Byte], once: Array[Byte]): Array[Byte] = {
```

Comment:
typo?

Reply:
Oh yeah... I remember writing `once` here and there instead of `nonce` 😅

Reply:
Fixed